mynearwallet/my-near-wallet
# Steve/tailwindcss

## Description

Allow developer to use TailwindCSS

## Test

- Run `yarn dev`
- Go to `https://localhost:1234/connection`, this page is written in tailwindCSS to make sure it functions properly.
